*: add specific Helm version (#1820)
pingcap/docs: dev/tidb-in-kubernetes/reference/tools/in-kubernetes.md

## Use Helm

> [Helm](https://helm.sh/) is a package management tool for Kubernetes. Currently we only support Helm version >= 2.9.0 and < 3.0.0. The installation steps are as follows:

Reviewer: ```suggestion
[Helm](https://helm.sh/) is a package management tool for Kubernetes. Make sure your Helm version >= 2.9.0 and < 3.0.0. The installation steps are as follows:
```

    curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash
    ```

>     Or if on macOS, you can use homebrew to install Helm via the command below:

Reviewer: ```suggestion
    Or if on macOS, you can use Homebrew to install Helm via the following command:
```
